### ISSUE TITLE
[NETSDKE2E]With NET8 RC2 installed, Warning: Failed to read or parse localization file "C:\Program Files\dotnet\templates\8.0.0-rc.2.23476.41\microsoft.dotnet.web.projecttemplates.8.0.8.0.0-rc.2.23476.41.nupkg(/content/BlazorWeb-CSharp/.template.config/localize/templatestrings.cs.json)".. displays during creating project.

### ISSUE THREAD

**Issue by vdanche** [dotnet/sdk#35768]
Reproduction Steps
1.install 8.0.100-rc.2.23477.19 SDK from [https://dev.azure.com/dnceng/internal/_build/results?buildId=2278108&view=artifacts&pathAsName=false&type=publishedArtifacts](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdev.azure.com%2Fdnceng%2Finternal%2F_build%2Fresults%3FbuildId%3D2278108%26view%3Dartifacts%26pathAsName%3Dfalse%26type%3DpublishedArtifacts&data=05%7C01%7Cv-huwang%40microsoft.com%7C35bc07f8b91d42d1a29708dbbfad8765%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638314524439333991%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=FiXctnEJOOASvzDsLPD3i4f%2BN%2B4p0HQPLO8%2B%2BHNi59g%3D&reserved=0)
2. In CLI, run "dotnet new console"
Expected behavior
console should be created successfully
Actual behavior
console can be created successfully, but below warning displays
![image](https://github.com/dotnet/installer/assets/65637796/d85b3765-517c-4ba0-8a36-b6e0658af38e)

dotnet --info:
.NET SDK:
Version:   8.0.100-rc.2.23477.19
Commit:    891192d164

Runtime Environment:
OS Name:     Windows
OS Version:  10.0.25398
OS Platform: Windows
RID:         win-x64
Base Path:   C:\Program Files\dotnet\sdk\8.0.100-rc.2.23477.19\

 

.NET workloads installed:
There are no installed workloads to display.

 

Host:
  Version:      8.0.0-rc.2.23475.17
  Architecture: x64
  Commit:       0b25e38ad3

 

.NET SDKs installed:
  8.0.100-rc.2.23477.19 [C:\Program Files\dotnet\sdk]

 

.NET runtimes installed:
  Microsoft.AspNetCore.App 8.0.0-rc.2.23476.41 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 8.0.0-rc.2.23475.17 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 8.0.0-rc.2.23476.8 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

 

Other architectures found:
  None

 

Environment variables:
  Not set

 

global.json file:
  Not found

 

Learn more:
https://aka.ms/dotnet/info

 

Download .NET:
https://aka.ms/dotnet/download

**Comment by yinjiehe01** [comment]
This issue also repro's on Mac.

**Comment by ChenhuiYuan01** [comment]
This issue also repro's on Linux.

**Comment by marcpopMSFT** [comment]
@MiYanni this is in the rc2 candidate build. It's not a blocker but it's noisy. Can you prioritize? Thanks.

**Comment by halter73** [comment]
@danroth27 filed https://github.com/dotnet/aspnetcore/issues/51003 in the aspnetcore repo to follow this. I left the comment there. We did just modify the files the SDK is complaining about, so it seems likely there's something wrong with them. Is it a problem that they're now only partially translated after some cross-release-branch merging?

**Comment by MiYanni** [comment]
@halter73 
> Is it a problem that they're now only partially translated after some cross-release-branch merging?

I did similar investigation to you. I looked at the RC1 files and `symbols/IncludeSampleContent/description` isn't translated either, so I don't think partial translations are to blame.

Based on your exception output in the linked issue, it would be one of these 2 lines from the templating code:
https://github.com/dotnet/templating/blob/63165401705fa7c7404ec7317c4178ff9769c484/src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/LocalizationModelDeserializer.cs#L114-L115

So, it clearly seems to think that either a `displayName` or a `description` is duplicated. But I don't see any duplications. That code hasn't been changed in over 2 years so it likely has something to do with the JSON files themselves.

**Comment by MiYanni** [comment]
The fix for this has been merged here: https://github.com/dotnet/templating/pull/7103

It will be part of the .NET SDK 8.0.100 RTM.

**Comment by vdanche** [comment]
@MiYanni This issue not repro on net SDK 8.0.100-rtm.23506.1(runtime-8.0.0-rtm.23504.8)
But it still repro on 8.0.200-preview.23509.1, keep tracking

**Comment by MiYanni** [comment]
Anything from 8.0.100 will flow into 8.0.200. Just might take some time.

**Comment by wtgodbe** [comment]
@MiYanni looks like we're still seeing this issue w/ the RTM SDK (`8.0.100-rtm.23506.1`), but only on Alpine: https://github.com/dotnet/aspnetcore/issues/51198

**Comment by MiYanni** [comment]
@wtgodbe

The issue you linked to, https://github.com/dotnet/aspnetcore/issues/51198, is a different issue than this one. This issue here is fixed.

The issue you linked to contains:
```
Warning: Locale 'pt-BR' does not match any known cultures.
Warning: Locale 'ru' does not match any known cultures.
Warning: Locale 'tr' does not match any known cultures.
Warning: Locale 'zh-Hans' does not match any known cultures.
Warning: Locale 'zh-Hant' does not match any known cultures.
Warning: Locale 'cs' does not match any known cultures.
Warning: Locale 'de' does not match any known cultures.
```

That error doesn't relate to this issue as this issue was faulty parsing of the content in the localization files. The culture error likely has a different root cause. The new error comes [from here](https://github.com/dotnet/templating/blob/7f810f92886dbbd017e9e0fdda9599ee5de8a646/src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/DirectoryBasedTemplate.cs#L141-L152), before anything is parsed. So, I'm closing this issue in favor of the new one.

**Comment by vdanche** [comment]
also fixed on 8.0.200-preview.23523.7(runtime-8.0.0-rtm.23520.16) SDK
